packages/lib-core/src/extensions/pages.ts

```diff
@@ -0,0 +1,63 @@
+import type { AnyObject } from '@monorepo/common';
+import type { ComponentType } from 'react';
+import type { RouteComponentProps } from 'react-router';
+import type { ExtensionK8sResourceIdentifier } from '../types/common';
+import type { CodeRef, Extension } from '../types/extension';
+
+/** Adds new page to Console router. */
+export type RoutePage = Extension<'core.page/route', RoutePageProperties>;
+
+/** Adds new resource list page to Console router. */
+export type ResourceListPage = Extension<
+  'core.page/resource/list',
+  ResourcePageProperties & AnyObject
+>;
+
+/** Adds new resource details page to Console router. */
+export type ResourceDetailsPage = Extension<
+  'core.page/resource/details',
+  ResourcePageProperties & AnyObject
```

Comment:
Let's nuke `& AnyObject` part for now, its original purpose was to make https://github.com/vega/ts-json-schema-generator happy in context of Console plugin SDK.

Reply:
Removing it.